server/src/com/thoughtworks/go/server/service/ArtifactsService.java

```diff
@@ -156,8 +161,14 @@ public void moveConsoleArtifacts(LocatableEntity locatableEntity) {
         }
     }
 
-    public File temporaryConsoleFile(JobIdentifier jobIdentifier) {
-        return chooser.temporaryConsoleFile(jobIdentifier);
+    public File temporaryConsoleFile(JobIdentifier jobIdentifier) throws IllegalArtifactLocationException {
```

Comment:
```temporaryConsoleFile``` is no longer just temporary now. If and when we move this out to a different service, we can think of a more appropriate name?